### PR TITLE
Feature/rosparam

### PR DESCRIPTION
**Pull request by alamarre3**
## Description
Generate a new build of proc_control with matlab 2022a

## Fixes
For the list of fixes, please check [control library](https://github.com/sonia-auv/control_library)

## How has this been tested ?
locally on my machine

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ ENV SONIA_WS=${SONIA_HOME}/ros_sonia_ws
 
 ENV NODE_NAME=${NODE_NAME}
 ENV NODE_PATH=${SONIA_WS}/src/${NODE_NAME}
-ENV LAUNCH_FILE=proc_control_node.launch
+ENV LAUNCH_FILE=proc_control.launch
```
Comment (Kam0ss) [inline Dockerfile:21]: This should be LAUNCH_FILE=${NODE_NAME}.launch

File: Dockerfile [hunk 2]
```diff
@@ -18,7 +18,7 @@ ENV SONIA_WS=${SONIA_HOME}/ros_sonia_ws
 
 ENV NODE_NAME=${NODE_NAME}
```
Comment (Kam0ss) [inline Dockerfile:19]: I think this line can be deleted

File: config/LOCAL.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+#MPC Gain Mode 11
+/proc_control/mpc/gains/c11/ov : '30,30,30,45,45,45,45,0,0,0,0,0,0' 
+/proc_control/mpc/gains/c11/mv : '0.2,0.2,0.2,0.2,0.2,0.2,0.2,0.2'
+/proc_control/mpc/gains/c11/mvr : '0.4,0.4,0.4,0.4,0.5,0.5,0.5,0.5'
+
+#MPC Gain Mode 19
+/proc_control/mpc/gains/c19/ov : '0,0,0,0,0,0,0,20,20,20,20,20,20'
+/proc_control/mpc/gains/c19/mv : '0.2,0.2,0.2,0.2,0.2,0.2,0.2,0.2'
+/proc_control/mpc/gains/c19/mvr : '0.1,0.1,0.1,0.1,0.3,0.3,0.3,0.3'
```
Comment (Kam0ss) [inline config/LOCAL.yaml:30]: Maybe we could have two LOCAL files, one for each sub ?

File: package.xml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <package>
-  <name>proc_control_node</name>
-  <version>1.3.0</version>
+  <name>proc_control</name>
+  <version>1.4.3</version>
   <description>
-  This package is generated from proc_control_node Simulink model.
+  This package is generated from proc_control Simulink model.
   </description>
   <maintainer email="rosuser@test.com">Les Alex</maintainer>
```
Comment (Kam0ss) [inline package.xml:7]: Change the adress for a valid one

**Review by Kam0ss — CHANGES_REQUESTED** [review]
Just a few changes to do

File: config/LOCAL.yaml (hunk 3 above)
Comment (alamarre3) [reply config/LOCAL.yaml:30]: Technically, the same config should be used locally as in the pool. Local Config is more a debug feature to leave the config that works in the pool intact

**Comment by alamarre3** [comment]
@Kam0ss I made change, but now the github action failed. Any idea ?

**Comment by Kam0ss** [comment]
> @Kam0ss I made change, but now the github action failed. Any idea ?

Try putting back NODE_NAME=${NODE_NAME}

**Comment by alamarre3** [comment]
@Kam0ss I fixed the problem. Gazebo msg dependencies were still in the cmakelist

**Review by Kam0ss — APPROVED** [review]
LGTM

**Pull request merged**